src/openforms/submissions/report.py

```diff
@@ -35,11 +65,13 @@ def show_payment_info(self) -> bool:
         return self.submission.payment_required and self.submission.price
 
     @property
-    def co_signer(self) -> str:
-        if not self.submission.co_sign_data:
-            return ""
+    def co_signer(self) -> CoSigner | None:
+        if not (co_sign_data := self.submission.co_sign_data):
+            return None
+
+        co_signer = CoSigner(co_sign_data)
```

Comment:
this should be something more along the lines of:

```py
co_signer = CoSigner(identifier=co_sign_data.get('identifier") or "", representation=co_sign_data.get("representatin") or "", ...)
```
-> do the normalization in this central entrypoint to make the dataclass strongly and completely typed instead of dealing with flexible types like dicts inside of them.